executor_manager/executors/docker/utils.py

```
@@ -479,5 +488,191 @@ def get_container_ports(container_name: str) -> dict:
+            created_at_str = parts[2]
+
+            # Parse creation time (Docker format: 2024-01-15 10:30:00 +0800 CST)
+            try:
+                # Simple parsing - just extract the timestamp
+                from datetime import datetime
+
+                # Docker uses format like "2024-01-15 10:30:00 +0800 CST"
+                # We'll parse the first 19 characters as datetime
+                created_dt = datetime.strptime(created_at_str[:19], "%Y-%m-%d %H:%M:%S")
+                container_age = current_time - created_dt.timestamp()
+            except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)